### PR TITLE
Added rawOutput param on StripeException object

### PR DESCRIPTION
**Pull request by alexrayan**
Added a way to pass raw output of Stripe exception via Handler to the Exception object as rawOutput parameter in order to handle exceptions that carry additional data like charge identifier, etc.

### CODE REVIEW

File: src/Utility.php [hunk 1]
```diff
@@ -56,9 +56,9 @@ protected static function needsAmountConversion(array $parameters)
         $currencies = [
             'BIF', 'DJF', 'JPY', 'KRW', 'PYG',
             'VND', 'XAF', 'XPF', 'CLP', 'GNF',
-            'KMF', 'MGA', 'RWF', 'VUV', 'XOF',
+            'KMF', 'MGA', 'RWF', 'VUV', 'XOF', 'USD',
```
Comment (brunogaspar) [inline src/Utility.php]: Don't add this, as this is a breaking change.

File: src/Utility.php [hunk 2]
```diff
@@ -56,9 +56,9 @@ protected static function needsAmountConversion(array $parameters)
...
         ];
 
-        return ! $hasCurrency || ($hasCurrency && ! in_array($parameters['currency'], $currencies));
+        return $hasCurrency && ! in_array($parameters['currency'], $currencies);
```
Comment (brunogaspar) [inline src/Utility.php]: Same here.

File: src/Utility.php (hunk 1 above)
Comment (alexrayan) [inline src/Utility.php]: Removed from the latest commit. I'll keep these changes in a branch to use in our local projects.

File: src/Utility.php (hunk 2 above)
Comment (alexrayan) [inline src/Utility.php]: The same here.

**Pull request merged**